src/intervals/configuration.jl

```diff
@@ -0,0 +1,57 @@
+"""
+Holds configuration information for the package:
+
+- `directed_rounding`:
+
+    Which method to use to implement **directed rounding**.
+    Possible values are:
+
+    - `:tight`: correct rounding using error-free arithmetic and CRlibm. Gives
+        the tightest resulting intervals, with a width of 1 ulp for each basic operation
+    - `:fast`: uses `prevfloat` and `nextfloat` to give a faster result, with width 2 ulps
+    - `:none`: no rounding; does *not* guarantee that the results are correct.
+
+- `powers`:
+    Method to implement powers. Possible values are
+    - `:tight`: gives tightest possible result by using `BigFloat` internally, and hence is slow
+    - `:fast`: uses repeated multiplication based on `Base.power_by_squaring`, and hence is faster but gives slightly looser results
+"""
+
+const configuration = Dict(:directed_rounding => :tight,
+                           :powers => :fast)
```

Comment:
I think we should have as default configuration that *both* defaults are `:tight`; It seems to me a bit unnatural to have `(:directed_rounding => :tight` and `:powers => :fast`. Note that tests change to this mode.